packs/data/equipment-effects.db/effect-rime-crystal-weapon-greater.json

```
@@ -0,0 +1,55 @@
+{
+    "_id": "YsV7tB15XrSCKNnB",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Rime Crystal - Weapon (Greater)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Rime Crystal (Greater)]{Rime Crystal (Greater)}</p>\n<p>After you cast a cold spell by activating the <em>crystal</em>, your Strikes with the weapon deal an additional 1d6 cold damage until the end of your next turn.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 8
+        },
+        "rules": [
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
+            },
+            {
+                "damageType": "cold",
+                "diceNumber": 1,
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "selector": "{item|flags.pf2e.rulesSelections.effectRimeCrystalWeaponGreater}-damage"
+            }
```

```suggestion
            {
                "choices": {
                    "ownedItems": true,
                    "types": [
                        "weapon"
                    ]
                },
                "flag": "weapon",
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
            },
            {
                "damageType": "cold",
                "diceNumber": 1,
                "dieSize": "d6",
                "key": "DamageDice",
                "selector": "{item|flags.pf2e.rulesSelections.weapon}-damage"
            }
```
We should prefer explicit flags.

Well, explicitly set flags. Not flags that swear, though honestly I'm fine with those too.